File: bootstrap/src/main/java/net/dontdrinkandroot/wicket/bootstrap/css/AlertStyle.java
package net.dontdrinkandroot.wicket.bootstrap.css;

import net.dontdrinkandroot.wicket.css.CssClass;


public enum AlertStyle implements CssClass {

	DANGER("alert-danger"),
	INFO("alert-info"),
	SUCCESS("alert-success"),
	WARNING("alert-warning");

	private String classString;


	private AlertStyle(String classString) {

		this.classString = classString;
	}


	@Override
	public String getClassString() {

		return this.classString;
	}
}

File: bootstrap/src/main/kotlin/net/dontdrinkandroot/wicket/bootstrap/css/AlertStyle.kt
package net.dontdrinkandroot.wicket.bootstrap.css

import net.dontdrinkandroot.wicket.css.CssClass

enum class AlertStyle(override val classString: String) : CssClass {
    PRIMARY("alert-primary"),
    SECONDARY("alert-secondary"),
    SUCCESS("alert-success"),
    DANGER("alert-danger"),
    WARNING("alert-warning"),
    INFO("alert-info"),
    LIGHT("alert-light"),
    DARK("alert-dark");
}